microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -183,43 +272,45 @@ For more information about the new visual **Font Editor**, see [Edit CSS font st
 
 This experiment is available starting with Microsoft Edge version 89.
 
-
-<!-- ====================================================================== -->
-<!-- ## Enable automatic contrast issue reporting via the Issues Panel -->
-<!-- present in 96, 98 -->
+For more information, see [Edit CSS font styles and settings in the Styles pane in DevTools](../inspect-styles/edit-fonts.md).
 
 
 <!-- ====================================================================== -->
-<!-- ## Enable experimental cookie features -->
+## Enable automatic contrast issue reporting via the Issues Panel
 <!-- present in 96, 98 -->
 
+Needs content.
 
 <!-- ====================================================================== -->
-<!-- ## Enable experimental hide issues menu -->
+## Enable experimental cookie features
 <!-- present in 96, 98 -->
 
+Needs content.
+
 
 <!-- ====================================================================== -->
-<!-- ## Allow grouping and hiding of issues by IssueKind -->
+## Enable experimental hide issues menu
 <!-- present in 96, 98 -->
 
+Needs content.
 
 <!-- ====================================================================== -->
-<!-- ## Enable Reporting API panel in the Application panel -->
+## Allow grouping and hiding of issues by IssueKind
 <!-- present in 96, 98 -->
 
+Needs content.
 
 <!-- ====================================================================== -->
-<!-- ## Enable CSS \<length\> authoring tool in the Styles pane (https://goo.gle/length-feedback) -->
-<!-- present in 96, 97, 98 -->
-<!-- in 97 & 98 the label is shorter: -->
-<!-- ## Enable CSS \<length\> authoring tool in the Styles pane -->
+## Enable Reporting API panel in the Application panel
+<!-- present in 96, 98 -->
 
+Needs content.
```

Comment:
Proposal:

"The Reporting API can be used to catch certain errors (like security violations or deprecated API calls) that occur when users visit your site and send them to a server endpoint. Enable this experiment to add the **Reporting API** section in the **Application** panel, which lists all of the reports sent to the endpoint."

Reply:
added